Python/tier2_typepropagator.c.h

```diff
@@ -412,7 +412,7 @@
         }
 
         TARGET(UNPACK_EX) {
-            STACK_GROW((oparg >> 8) + (oparg & 0xFF));
+            STACK_GROW((oparg & 0xFF) + (oparg >> 8));
```

Comment:
Curious: why did this change?

Reply:
Ah crap forgot to regen. Regen would produce identical results.

It doesn't matter though, this happened because I reversed the order of `self.output_effects` in preparation for "processing" each one in reverse order since that's what the DSL assumes. To write the `stack_adjust` part I used the reversed version initially. Then for consistency I switched to the non-reversed one.

Reply:
Okay could you regen please? Thanks!

Reply:
Okie done